nss-day-cohort-48/rare-api-orchestra-outlaws
# Reactions: GET all, DELETE

## Testing:
1. Pull down from `al-reactions` branch
2. Make migrations, migrate, load data from reactions.json
3. Generate superuser/auth token in `localhost:8000/admin` view
4. Ensure that GET at (/reactions) works for all
5. Ensure that DELETE at (/reactions/#) works
